src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -84,67 +104,76 @@ const AnnotationActivity = ({
-                    {isEditing && currentUser ? (
-                        <CommentForm
-                            className="bcs-AnnotationActivity-editor"
-                            entityId={id}
-                            getAvatarUrl={getAvatarUrl}
-                            getMentionWithQuery={getMentionWithQuery}
-                            isEditing={isEditing}
-                            isOpen={isEditing}
-                            mentionSelectorContacts={mentionSelectorContacts}
-                            onCancel={handleFormCancel}
-                            updateComment={handleFormSubmit}
-                            user={currentUser}
-                            tagged_message={message}
+                            isDisabled={isFileVersionUnavailable}
+                            message={activityLinkMessage}
+                            onAnnotationSelect={handleOnSelect}
                         />
-                    ) : (
-                        <ActivityMessage id={id} tagged_message={message} getUserProfileUrl={getUserProfileUrl} />
-                    )}
-                    <AnnotationActivityLink
-                        data-resin-iscurrent={isCurrentVersion}
-                        data-resin-itemid={id}
-                        data-resin-target="annotationLink"
-                        id={id}
-                        isDisabled={isFileVersionUnavailable}
-                        message={activityLinkMessage}
-                        onClick={handleOnSelect}
-                    />
-                </Media.Body>
-            </Media>
-            {/* $FlowFixMe */}
-            {error ? <ActivityError {...error} /> : null}
-        </ActivityCard>
+                        {isEditing && currentUser ? (
+                            <CommentForm
+                                className="bcs-AnnotationActivity-editor"
+                                entityId={id}
+                                getAvatarUrl={getAvatarUrl}
+                                getMentionWithQuery={getMentionWithQuery}
+                                isEditing={isEditing}
+                                isOpen={isEditing}
+                                mentionSelectorContacts={mentionSelectorContacts}
+                                onCancel={handleFormCancel}
+                                updateComment={handleFormSubmit}
+                                user={currentUser}
+                                tagged_message={message}
+                            />
+                        ) : (
+                            <ActivityMessage id={id} tagged_message={message} getUserProfileUrl={getUserProfileUrl} />
+                        )}
+                    </Media.Body>
+                </Media>
+                {/* $FlowFixMe */}
+                {error ? <ActivityError {...error} /> : null}
+            </SelectableActivityCard>
+            {isMenuVisible && (
+                <AnnotationActivityMenu
```

Comment:
It seems a little odd that this is totally outside the parent card. Can this stay within the Media body? If not, should it be moved to a parent component?

Reply:
There's a Media.Menu component which ideally would be compatible with this use case. It goes inside the Media as a sibling of Media.Body

Reply:
As discussed, the reason this menu has to live outside is to avoid event bubbling issues for events that occur within the dropdown menu which interferes with the `mousedown` event processing that is down to power the deselect annotation logic

Reply:
Is there something we can fix about that in the Media element? I think in other parts of the app we added `evt.stopPropagation()` to some of the Media menu item event handlers to address some of that.

Reply:
Yeah @jstoffan it's related to the `mousedown` handler in `AnnotationActivity`. If the menu is a child of `SelectableActivityCard` then there are issues when the card is selected and the menu is opened -- the `mousedown` is prevented from propagating up to the logic that would deselect the annotation. 

@alexkrolick I'm not sure, this use case is very specific to how annotations works and I had done a quick search and there were only 1 or 2 other instances where these types of event handling occurred so maybe it's not widely an issue